app/src/pages/outside/registrationPage/registrationForm/registrationForm.jsx

```diff
@@ -84,20 +84,30 @@ export class RegistrationForm extends Component {
     autofill: PropTypes.func.isRequired,
     email: PropTypes.string,
     loading: PropTypes.bool,
+    initData: PropTypes.object,
   };
 
   static defaultProps = {
     submitForm: () => {},
     email: '',
     loading: false,
+    initData: {},
```

Comment:
`initData` sounds like an action, pls rename it to `initialData` or smth like that.